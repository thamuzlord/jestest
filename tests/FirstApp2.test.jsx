import { render, screen } from "@testing-library/react"
import { FirstApp } from "../src/FirstApp"

describe('Pruebas de componente', () => {

    const title = 'Hola VASCO'

    test('match snapshot', () => { 
        const {container} = render(<FirstApp title={title} />)
        expect(container).toMatchSnapshot()
     })

     test('show "Hola VASCO"', () => { 
        render(<FirstApp title={title} />)
        expect(screen.getByText(title))
      })

      test('debe estar en un h1', () => { 
        render(<FirstApp title={title} />)
        expect(screen.getByRole('heading', {level: 1}).innerHTML).toContain(title)
       })

})