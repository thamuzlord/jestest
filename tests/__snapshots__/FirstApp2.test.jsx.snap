// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de componente match snapshot 1`] = `
<div>
  <h1
    data-testid="test-title"
  >
     
    Hola VASCO
     
  </h1>
  <p>
    No hay subt√≠tulo
  </p>
  <p>
    Fernando Herrera
  </p>
</div>
`;
